selfdrive/car/hyundai/carstate.py

```diff
@@ -104,8 +104,8 @@ def update(self, cp, cp_cam):
       # These are not used for engage/disengage since openpilot keeps track of state using the buttons
       ret.cruiseState.available = cp.vl["TCS13"]["ACCEnable"] == 0
       ret.cruiseState.enabled = cp.vl["TCS13"]["ACC_REQ"] == 1
-      ret.cruiseState.standstill = False
-      ret.cruiseState.nonAdaptive = False
+      ret.cruiseState.standstill = cp_cruise.vl["SCC11"]["SCCInfoDisplay"] == 4.
```

Comment:
This was also updated to match stock, assumption is that since main was never used to make ACC available there was no need to nonAdaptive or standstill set to a value.